Makefile

```diff
@@ -35,4 +29,4 @@ build: generate fmt vet ## Build manager binary.
 
 .PHONY: clean
 clean:
-	@rm -rf bin/ pkg/protos/*.pb.go
+	@rm -rf bin/
```

Comment:
Since the `build` command depends on `generate` previously, it should also be modified.